### DISCUSSION TITLE
Regarding Filter Feature

### DISCUSSION THREAD

**Discussion by vikasphougat** [dailydotdev/daily#924] [Feature Requests]
hello daily dev Team 

my concern is about the filter functionality , which is only implemented in <Most Voted> component .

1- can you apply same filter functionality on <My feed > and <Popular>  components.
2- add two more filter value for TODAY and YESTERDAY ....LAST WEEK , LAST MONTH ,, LAST YEAR ,, is already there.

idea behind this change is , regular reader get very old post , we have to scroll down a lot to find today and yesterday updated blog or post .

Thanks 
Vikas

**Comment by idoshamun** [comment]
Hi @vikasphougat,

Just to make sure I understand, are you referring to the time filter dropdown in the most upvoted feed? You wish to apply it to the "my feed" as well?

**Reply by vikasphougat** [reply]
yes , if it is possible ?,, because on my feed page ,,there is no filtering by date , by default on my feed shows random article or news , 
if there is a way to filter those by date ,, it will be very helpful  for user .
